it/test/controllers/agentAppointment/AppointAgentsControllerISpec.scala

```diff
@@ -776,18 +781,287 @@ class AppointAgentsControllerISpec extends ISpecBase with HtmlComponentHelpers {
         document.select(continueButtonLocator).text shouldBe continueButtonTextWelsh
       }
     }
+
+    "display the correct search content in English - manageAgentJourney true" which {
```

Comment:
For all the English tests you're missing the word 'error' in the test name (like you have in Welsh). Please can you add it just for clarity